cmd/cmd.go

```diff
@@ -71,7 +72,12 @@ func Execute() {
 			logger.Fatalf("couldn't convert data to logical indexes: table %s. reason: %s", tableName, err)
 		}
 
-		_, err = markdown.WriteString(format.CreateTableMarkdown(tableName, table.Comment, columns, logicalIndexes))
+		foreignKeyData, err := format.GetForeignKeyDescriptions(ctx, db, tableName)
+		if err != nil {
+			logger.Printf("couldn't query database to fetch foreign key data: table %s. reason: %s", tableName, err)
```

Comment:
hmmm. won't this make the output non-deterministic @mingjianliu ? what is going on here?

should we error out hard in this case?

Reply:
yeah I figure out what's broken and made a fix for it.
TL:DR is sqlx doesn't correctly understand the reflection of fields, so we need to make it more explicit

Reply:
ty